Dockerfile

```diff
@@ -17,6 +17,8 @@ RUN cd /opt && \
 # and hosts.
 RUN yum install -y nss_wrapper && yum clean all
 
+RUN yum install -y numpy && yum clean all
```

Comment:
ok, if there was a specific reason for separating this, then i don't have an issue. thanks for the explanation.